modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java

```diff
@@ -106,7 +106,7 @@ public List<DropdownItem> getActionDropdownItems() {
 						!PropsValues.PORTAL_JAAS_ENABLE &&
 						PropsValues.PORTAL_IMPERSONATION_ENABLE &&
 						(_user.getUserId() != _themeDisplay.getUserId()) &&
-						!_themeDisplay.isImpersonated() &&
+						_user.isActive() && !_themeDisplay.isImpersonated() &&
```

Comment:
We need to alphabetically sort these conditions. If you just move it in front of `!_themeDisplay.isImpersonated()` that should do it.

Reply:
Hi @stian-sigvartsen 

It's already in front of `!_themeDisplay.isImpersonated()`, isn't it?

Reply:
Sorry, my bad! Perfect then!